extensions/rvadapter/src/main/java/com/google/android/agera/rvadapter/RepositoryAdapter.java

```diff
@@ -128,7 +133,101 @@ public RepositoryAdapter build() {
       return new RepositoryAdapter(this);
     }
 
+    /**
+     * Builds the {@link RepositoryAdapter} that presents the provided repositories in order and
+     * observes the repositories as well as any additional observables while the provided
+     * {@link Activity} is resumed (between onResumed/onPaused).
+     */
+    @TargetApi(14)
+    @NonNull
+    public Adapter<ViewHolder> whileResumed(@NonNull final Activity activity) {
+      final RepositoryAdapter repositoryAdapter = new RepositoryAdapter(this);
+      activity.getApplication().registerActivityLifecycleCallbacks(
+          new AdapterActivityLifecycleCallbacks(activity){
```

Comment:
actually.... this anonymous subclass retains the Builder unnecessarily; maybe create standard `static` subclasses for these two cases